components/dashboards-web-component/src/designer/components/PagesSidePane.jsx

```diff
@@ -119,6 +125,9 @@ class PagesSidePane extends Component {
             }
             const currentLandingPage = this.props.dashboard.landingPage;
             this.props.dashboard.landingPage = newLandingPageId;
+            // Remove hidden flag from the new landing page
+            var page = this.getPageById(newLandingPageId);
+            delete page.hidden;
```

Comment:
In the UI we do not prevent user to make a hidden page the landing page. That means, even though the page is marked as hidden, we show the option to make that the home page. What this logic does is, when the user makes a hidden page the landing page we remove its hidden property making it visible in the menu. Hope this is what you expect in your comment as well?

BTW, instead of deleting we can set `page.hidden = false;`.

Reply:
> In the UI we do not prevent user to make a hidden page the landing page. That means, even though the page is marked as hidden, we show the option to make that the home page. What this logic does is, when the user makes a hidden page the landing page we remove its hidden property making it visible in the menu. Hope this is what you expect in your comment as well?

How about we
- disable the `Hide` checkbox if `Mark as home` is checked
- disable `Mark as home` checkbox if `Hide` is checked
This approch clearly convey that the landing page cannot be hidden and a hidden page cannot be the landing page.

>BTW, instead of deleting we can set page.hidden = false;
yes, lets do that.